src/styles.variables.scss

```diff
@@ -0,0 +1,4 @@
+@import '~materialize-css/sass/components/variables';
+
+$parallax-min-height: 840px !default;
+$section-min-height: 500px !default;
```

Comment:
These variables are not needed. The first one is applied to a component with static height and the second one is not being used anywhere. You could simply get rid of this file and rename `styles.colors.scss` -> `styles.variables.scss`. So we only have one file with style variables. Like that we also avoid importing materialize styles twice.